source/acquaman/AMStepScanActionController.cpp

```diff
@@ -169,7 +169,7 @@ void AMStepScanActionController::buildScanController()
 			scan_->addRawDataSource(new AMRawDataSource(scan_->rawData(), scan_->rawData()->measurementCount()-1), oneDetector->isVisible(), oneDetector->hiddenFromUsers());
 	}
 
-	if (scan_->rawData()->scanAxesCount() >= 2)
+	if (scan_->rawData()->scanAxesCount() >= 2 || (scan_->rawData()->scanAxesCount() == 1 && stepConfiguration_->direction() == AMScanConfiguration::Decrease))
```

Comment:
No, increasing scans don't prefill their data.  Scanning backwards needs to have it's data prefilled due to a limitation of the CDF data store.